wpilibNewCommands/src/test/native/cpp/frc2/command/CommandDecoratorTest.cpp

```diff
@@ -383,6 +383,43 @@ TEST_F(CommandDecoratorTest, RaceWithOrder) {
   EXPECT_TRUE(firstWasPolled);
 }
 
+TEST_F(CommandDecoratorTest, Repeatedly) {
+  CommandScheduler scheduler = GetScheduler();
+
+  int counter = 0;
+
+  auto command = InstantCommand([&counter] { counter++; }, {}).Repeatedly();
+
+  scheduler.Schedule(command);
+
+  for (int i = 1; i <= 50; i++) {
+    scheduler.Run();
+    EXPECT_EQ(i, counter);
+  }
+
+  EXPECT_TRUE(scheduler.IsScheduled(command));
+}
+
+TEST_F(CommandDecoratorTest, RepeatedlyCount) {
+  CommandScheduler scheduler = GetScheduler();
+
+  int counter = 0;
+
+  auto command = InstantCommand([&counter] { counter++; }, {}).Repeatedly(3);
+
+  scheduler.Schedule(command);
+  EXPECT_TRUE(scheduler.IsScheduled(command));
+  for (int i = 1; i < 3; i++) {
+    scheduler.Run();
+    EXPECT_EQ(i, counter);
```

Comment:
Add the failure messages from Java with `<<`:

```cpp
EXPECT_EQ(i, counter) << "message";
```

Reply:
added in d968cb3